src/main/java/javaslang/collection/CharSeq.java

```diff
@@ -386,6 +386,18 @@ public CharSeq intersperse(Character element) {
         return result;
     }
 
+    @Override
+    public CharSeq padTo(int length, Character element) {
+        if(length <= back.length()) {
+            return this;
+        }
+        final StringBuilder sb = new StringBuilder(back);
+        for (int i = 0; i < length - back.length(); i++) {
```

Comment:
just a little thing: let's introduce a `final int bound = length - back.length();` that saves us calculating the bound each time checking the loop condition.